### DISCUSSION TITLE
data files for Cowpertwait & Metcalf

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#2] [Q&A] [answered]
I don't know if anyone else is trying to follow along with the exercises in the CW book, but the links provided no longer work. If you download the supplementary material from their book (https://extras.springer.com/?query=978-0-387-88697-8) you will find the data needed under webfiles.

**Comment by eeholmes** [comment, marked as answer]
Thanks! I added a link to the suppl info to the cover of the pdf.
